Story1/task/make 30 default

Liliya-V opened the pull request.

ailindoyle commented on app/js/checkAnswers.js:
```diff
@@ -60,17 +60,40 @@ function getUserAnswers(questionAmount) {
+//             console.log(`Looks like there was an error: ${err}`)
+//     })
+// }
+
```
need to get rid of commented out code

marty-crane replied:
Resolved

ailindoyle commented on app/adminPage.html:
```diff
@@ -18,6 +18,7 @@
     <script src="js/validateUserAdditions.js" defer></script>
     <script src="js/ajaxHandlebarsTemplate.js" defer></script>
     <script src="js/handlebarsUserTable.js" defer></script>
+    <script src="js/ajaxHandlebarsTemplate.js" defer></script>
```
this file is already included on line 19

marty-crane replied:
resolved

Liliya-V committed:
Validated a time input so its not zero ro null or empty,
Made 30 a default,
Made is to be sent to JSOn timed 60 so its seconds in a DB ( not minutes)

Liliya-V committed:
Merge branch 'Story1/task/time-input-field' into Story1/task/make-30-default

Liliya-V committed:
new user form validation

rossanderson75 commented on app/js/validateUserAdditions.js:
```diff
@@ -85,18 +85,25 @@ document.querySelector('#addNewUserForm').addEventListener('submit', function(ev
 
     getExistingUsers().then(function(existingUsers) {
         if (!isEmailValid(emailField.value) || userExists(emailField.value, existingUsers)) {
-            var errorMessage = "Your email is not valid or already exists: Please provide a correct email"
```
Once I corrected the email error, the error message still appeared.

rossanderson75 commented on app/js/validateUserAdditions.js:
```diff
@@ -85,18 +85,25 @@ document.querySelector('#addNewUserForm').addEventListener('submit', function(ev
-            var errorMessage = "Your email is not valid or already exists: Please provide a correct email"
-            errorField.innerHTML = errorMessage
+            errorField.innerHTML = "Your email is not valid or already exists: Please provide a correct email"
+        } else if (timeField.value <=1 || timeField.value == null) {
```
Please can we check if numeric or not a number.

rossanderson75 reviewed (commented):
Made a couple of suggestions to improve validation

Liliya-V committed:
made error message disappeare after fixing email,
done a valitation on the input being a number
